src/pages/Inventory.page.js

```diff
@@ -9,7 +9,49 @@ export class InventoryPage extends BaseSwagLabPage {
 
     get addItemToCartBtns() { return this.page.locator('[id^="add-to-cart"]'); }
 
+    inventoryItemsName = '.inventory_item_name';
+
+    inventoryItemsPrice = '.inventory_item_price';
+
+    get sort() { return this.page.locator('.product_sort_container'); }
+
+    get activeOption() { return this.page.locator('.active_option'); }
+
+    async newSortItem(value) {
```

Comment:
Let's name this method to be self explained.
E.g. sortItemsBy, performSort, etc